Routing Directorate Early Review (#137)
FabioPeruzzini/actn-poi: draft-ietf-teas-actn-poi-applicability.md

This document uses packet-based Traffic Engineered (TE) service
examples. These are described as "TE-path" in this document. Unless
otherwise stated, these TE services may be instantiated using
Resource Reservation Protocol (RSVP) Traffic Engineering (TE)-based or
Segment Routing Traffic Engineering (SR-TE)-based, forwarding plane
mechanisms.

This document assumes familiarity with the ACTN architecture defined in

> {{!RFC8453}}, the TE topology model defined in {{!RFC8795}}, and the

Reviewer: Isn't the syntax for references relying on question marks instead of exclamation marks?

Reviewer 2: I'll fix that in the final version. Thanks J.

Author: The kramdown syntax for referencing IETF I-Ds/RFCs in line uses the exclamation mark for normative references and the question mark for informative references

Therefore {{!RFC8453}} is a normative reference to RFC8453

It is possible to reference the same I-D/RFC multiple times mixing informative and normative types: in this case, kramdown compiler will take care to add that I-D/RFC only in the normative references section. If, for any reason, all the text fragments with normative references to that I-D/RFC are removed, kramdown compiler will take care to move that I-D/RFC to the informative references section

- The L2NM and L3NM YANG data models, defined in {{?RFC9291}} and
{{?RFC9182}}, whose primary focus is the MPI, can also be used to

> provide L2VPN and L3VPN network service configuration from a

Reviewer: It seems to me "an orchestrated..." was correct. Am I missing something?

Reviewer 3: I agree..."an orchestrated.." seems correct

Author: ```suggestion
provide L2VPN and L3VPN network service configuration from an
```

   to the LAG then original Bandwidth is recovered between the end
   routers.

>    Note: in this LAG scenario let assume that BFD is running at LAG

Reviewer: Not sure about the syntax of that sentence...

Author: ```suggestion
   Note: in this LAG scenario it is assumed that BFD is running at LAG
```

   level so that there is nothing triggered at MPLS level when one of
   the link member of the LAG fails.

> 1. If there is no LAG then the scenario is not clear since a IP router

Reviewer: Same here: the former "an IP router" read correct to me...

Reviewer 2: It looks like something went wrong and replaced proposed/corrected text. Maybe we had multiple branches and "merge incoming" instead of "accept incoming" was performed. Either way, I'll do a sweep of all changes on final version.

Author: ```suggestion
1. If there is no LAG then the scenario is not clear since an IP router
```
